integration/spark-common/src/main/scala/org/apache/carbondata/spark/util/CommonUtil.scala

```diff
@@ -369,6 +369,14 @@ object CommonUtil {
     }
   }
 
+  def validateListInfo(listInfo: List[List[String]]): Unit = {
```

Comment:
simplify this by
```
val list = listInfo.flatted
if (list.length != list.distinct.size) {
  sys.error("xxx")
}
``

Reply:
Hi @xuchuanyin ,
Thank you for the comments. Have changed to use flatten function.
1. Put in CommonUtil not only for reusing but also keep code in one style and put similar function in one place.
2. distinct is case sensitive